### DISCUSSION TITLE
23. GET과 POST의 차이점에 대해 설명해 주세요.

### DISCUSSION THREAD

**Discussion by cutehumanS2** [jewelry-box/cs-interview#5] [General]
GET과 POST의 차이점에 대해 설명해 주세요.

**Comment by linirini** [comment]
서버에서 자원의 표현을 조회한 데 사용하는 GET 메서드는 웹 페이지를 요청할 때 주로 사용한다. 요청한 자원의 URL을 통해 필요한 정보를 명시하고, 서버가 해당 자원의 내용을 응답으로 반환한다. GET 메서드는 여러 번 요청해도 같은 결과를 반환하므로, 멱등성을 보장한다.
반면, POST 메서드는 서버에 특정 작업을 처리하도록 요청하는 메서드이다. POST는 동일한 요청을 여러 번 보내면 리소스가 중복 생성될 수 있으므로, 멱등성을 보장하지 않는다.
또한, POST 요청의 경우 메세지 본문이 사용되는 경우가 많지만, GET 요청의 경우에는 메세지 본문이 잘 사용되지 않는다.
